ci-operator/config/redhat-appstudio/rhtap-installer/redhat-appstudio-rhtap-installer-main.yaml

```diff
@@ -43,17 +43,63 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
-- as: rhtap-installer-static-checks
-  steps:
-    test:
-    - as: shellcheck
-      commands: |
-        #!/usr/bin/env bash
-        ls -l
-      from: rhtap-installer
-      resources:
-        requests:
-          cpu: 100m
+- as: rhtap-installer-shellcheck
+  commands: |
+    #!/usr/bin/env bash
+
+    set -o errexit
+    set -o nounset
+    set -o pipefail
+
+    find . -type f -name "*.sh" -print0 | xargs -r -0 shellcheck
+  container:
+    from: rhtap-installer
+- as: rhtap-installer-yamllint
+  commands: |
+    #!/usr/bin/env bash
+
+    set -o errexit
+    set -o nounset
+    set -o pipefail
+
+    CONFIG_FILE=yamllint.yaml
+    cat <<EOF > "$CONFIG_FILE"
+    ---
+    yaml-files:
+      - '*.yaml'
+      - '*.yml'
+    rules:
+      braces: enable
+      brackets: enable
+      colons: enable
+      commas: enable
+      comments:
+        require-starting-space: true
+        ignore-shebangs: true
+        min-spaces-from-content: 1
+      comments-indentation: enable
+      document-end: disable
+      document-start: enable
+      empty-lines: enable
+      empty-values: disable
+      float-values: disable
+      hyphens: enable
+      indentation: enable
+      key-duplicates: enable
+      key-ordering: disable
+      line-length: disable
+      new-line-at-end-of-file: enable
+      new-lines: enable
+      octal-values: disable
+      quoted-strings: disable
+      trailing-spaces: enable
+      truthy:
+        check-keys: false
+    EOF
+
+    yamllint -c "$CONFIG_FILE"
```

Comment:
This change will error out with because you have not aaded file dir which is pwd in [yamllint config file](https://github.com/openshift/release/blob/master/ci-operator/step-registry/redhat-appstudio/rhtap-installer/static-checks/yamllint/redhat-appstudio-rhtap-installer-static-checks-yamllint-commands.sh#L7C1-L45C27)
```
usage: yamllint [-h] [-] [-c CONFIG_FILE | -d CONFIG_DATA]
                [-f {parsable,standard,colored,github,auto}] [-s]
                [--no-warnings] [-v]
                [FILE_OR_DIR [FILE_OR_DIR ...]]
yamllint: error: one of the arguments FILE_OR_DIR - is required
{"component":"entrypoint","error":"wrapped process failed: exit status 2","file":"k8s.io/test-infra/prow/entrypoint/run.go:84","func":"k8s.io/test-infra/prow/entrypoint.Options.internalRun","level":"error","msg":"Error executing test process","severity":"error","time":"2024-02-16T[12](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/48881/rehearse-48881-pull-ci-redhat-appstudio-tssc-sample-pipelines-main-rhtap-sample-pipelines-yamllint/1758471702678867968#1:build-log.txt%3A12):43:19Z"}
```

Reply:
fixed. thank you so much to point out this issue